packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -741,12 +677,12 @@ export class LitNodeClientNodeJs
     if (!authSig) {
       throw new Error('authSig or sessionSig is required');
     }
-    const data: JsonExecutionRequest = {
-      authSig,
-      code,
-      ipfsId,
-      jsParams,
-      authMethods,
+    const data: JsExecutionRequestBody = {
+      ...(authSig ? { authSig } : {}),
+      ...(code ? { code } : {}),
+      ...(ipfsId ? { ipfsId } : {}),
+      ...(authMethods ? { authMethods } : {}),
+      ...(jsParams ? { jsParams } : {}),
     };
```

Comment:
Just curious why we needed to change this to use this pattern

Reply:
I was checking if the node would accept the request if a key is undefined or when it's omitted in the new endpoint. but i have created a separate interface for the request body in another pr

Reply:
Can we do a constructor type pattern or something similar which set the default values if none is provided?